controller/tests/test_admin_controller.py

```diff
@@ -0,0 +1,21 @@
+import unittest
+
+from controller.admin_controller import view_admin_profile
+from models.admin import Admin
+
+
+class AdminControllerTest(unittest.TestCase):
+
+    def setUp(self) -> None:
+        Admin.delete_all()
+
+    def test_profile(self):
+        user = Admin('test_user', 'test_password')
+        self.assertEqual(str(user), view_admin_profile(user))
+
+    def tearDown(self) -> None:
+        Admin.delete_all()
```

Comment:
I'm not clear what the difference between this function and def setUp is, as the functionality written seems to be identical. Can you please clarify?

Reply:
This question also applies to test_volunteer.py where the equivalent pair of functions also appear.

Reply:
`setUp` and `tearDown` are specific method names in the [`unittest` library](https://docs.python.org/3/library/unittest.html#basic-example). 
`setUp` got run before each test. 
`tearDown` got run after each test. 
They are the same because I want to clean the data both pbefore and after tests.

Reply:
Understood, thank you!